Installing-and-Configuring/OS/index.md

```diff
@@ -79,17 +79,34 @@ This feature may be turned off during configuration of heads.  The following wil
 
 ### Using only a LUKS passphrase
 
-In this model heads is not responsible for any of the encryption of data on disk.  The linux kernel decrypts and mounts LUKS containers by asking the user for a passphrase.  The correct passphrases allows boot to finish.
+In this model heads is not responsible for any of the decryption of data on disk.  The linux kernel decrypts and mounts LUKS containers by asking the user for a passphrase.  The correct passphrases allows boot to finish.
```

Comment:
In this model heads is not responsible for any of the decryption of data on disk. The linux kernel decrypts and mounts LUKS containers by asking the user for a passphrase. The correct passphrases allows boot to finish.

WRONG

Under the previous Disk Unlock Key scenario, Heads is not responsible to unlock Disk Unlock Key. It actually creates an additional initrd file pushed to QubesOS so that the OS is using that key to decrypt content without further user interaction.

In the Disk Recovery Key scenario, Heads doesn't attempt to release the Disk Unlock Key. It simply relies on OS mechanisms to prompt the user for the Disk Recovery Key passphrase at standard OS prompt.

Reply:
WIP

I would argue that since heads creates the key and initrd which is used by Qubes to decrypt data that heads is responsible even though it doesn't do the decryption step itself.  What if I add a section for Disk REcovery Key and rephrase this section ?

push in a minute

Reply:
This has been rewritten and needs review.  I'm not certain what exactly was WRONG with the first draft.  It now starts at line 86 in  OS/index